google/resource_container_cluster.go

```diff
@@ -1145,7 +1145,11 @@ func resourceContainerClusterUpdate(d *schema.ResourceData, meta interface{}) er
 	}
 
 	if d.HasChange("resource_labels") {
-		resourceLabels := d.Get("resource_labels").(map[string]string)
+		resourceLabelsI := d.Get("resource_labels").(map[string]interface{})
```

Comment:
Think we have this in `utils.go` - `convertStringMap`.

https://github.com/terraform-providers/terraform-provider-google/blob/master/google/utils.go#L286